lib/ansible/config/base.yml

```diff
@@ -1713,7 +1724,7 @@ PARAMIKO_LOOK_FOR_KEYS:
   type: boolean
 PERSISTENT_CONTROL_PATH_DIR:
   name: Persistence socket path
-  default: ~/.ansible/pc
+  default: '{{C.ANSIBLE_HOME}}/pc'
```

Comment:
`default: '{{ANSIBLE_HOME}}'` already works

Reply:
It doesn't for me, when trying that I can see in the output that the path used would be `{pwd}/{{ANSIBLE_HOME}}/pc` for this case.

Reply:
my bad, must start and END with moustaches:
```default: '{{ANSIBLE_HOME + "/pc"}}'`